lakeroad-egglog/tests/egglog_tests.rs

```diff
@@ -113,6 +113,43 @@ fn create_rewrites(
     for expr in &exprs {
         println!("{}", expr);
     }
+    /// Given an expression, like `(Var "i_out" 1)` - create a replacement map that replaces that with `i_out`
+    /// You can pass the replacement map returned by this function to replace_in_expr
```

Comment:
Could you add a [doctest](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html) to this?

e.g.
```suggestion
    /// Given an expression, like `(Var "i_out" 1)` - create a replacement map that replaces that with `i_out`
    /// You can pass the replacement map returned by this function to replace_in_expr
    /// ```
    /// <some code that shows what this function does, with assert statements to check the output>
    /// ```
```